packages/tangle/messagefactory.go

```diff
@@ -344,8 +346,12 @@ func PrepareReferences(strongParents MessageIDs, issuingTime time.Time, tangle *
 		opinionCanBeExpressed := true
 		for strongParentBranchID := range strongParentBranchIDs {
 			referenceParentType, referenceMessageID, err := referenceFromStrongParent(tangle, strongParentBranchID, issuingTime)
+			// Explicitly ignore error since we can't create a like/dislike reference to the message.
+			// We need to simply ignore it and remove it from the strong parents.
 			if err != nil {
-				return nil, errors.Errorf("failed to determine valid reference from Branch with %s: %w", strongParentBranchID, err)
+				referenceNotPossible.Add(strongParent)
```

Comment:
A code bit is missing no?

Later if `if !opinionCanBeExpressed {` we should introduce this strong parent as a weak reference if we like all its payload branches (we already do this bit) and in any case drop the message from the tippool and add its strong parents if they are not beyond `maxParentAge`.

Reply:
Handled the return now in a new event and attaching to it in the `TipManager`